botocore/utils.py

```diff
@@ -1085,3 +1085,54 @@ def _get_response(self, full_url, headers, timeout):
 
     def full_url(self, relative_uri):
         return 'http://%s%s' % (self.IP_ADDRESS, relative_uri)
+
+
+def get_configured_signature_version(service_name, client_config,
```

Comment:
Would it be possible to add it as a client method? I would prefer we do not add functionality to utils if it is not needed to be there and be public.